mongo/integration/mtest/mongotest.go

```diff
@@ -526,7 +526,10 @@ func (t *T) ClearCollections() {
 				// could prevent it from being dropped for sharded clusters. We can resolve this by
 				// re-instantiating the collection with a majority write concern before dropping.
 				collname := coll.created.Name()
-				wcm := writeconcern.New(writeconcern.WMajority(), writeconcern.WTimeout(1*time.Second))
+				wcm := &writeconcern.WriteConcern{
```

Comment:
Can we use the convenience function in the example, instead of using the "majority" magic string? 

```go
wcMajority := writeconcern.Majority()
wcMajority.WTimeout = 1 * time.Second
```